### PR TITLE
process.exit on close

### PR DESCRIPTION
**Pull request by anandkumarpatel**
add docker timeout
do not report 404
exit on errors

### CODE REVIEW

File: configs/.env [hunk 1]
```diff
@@ -4,6 +4,7 @@ EVENTS_SOCKET_TIMEOUT=0
 LOG_LEVEL=trace
 LOG_LEVEL_STDOUT=trace
 PORT=3000
+SWARM_TIMEOUT_MS=1000
```
Comment (podviaznikov) [inline configs/.env]: potentially seems too small to me

File: lib/datadog.js [hunk 2]
```diff
@@ -4,39 +4,17 @@
...
-  } else {
-    callback()
-  }
-}
-
-exports.stream = DatadogStream
-
-var inc = exports.inc = function (eventName) {
-  monitor.increment(eventName, 1, exports.tags)
+exports.timer = function (timerName, hostIp, orgId) {
```
Comment (podviaznikov) [inline lib/datadog.js:22]: if you deleted datadog stream - then just delete tis whole file and use `monitor-dog` directly

Comment (anandkumarpatel) [inline lib/datadog.js:22]: I like to keep it around to add common tags, unless you feel strongly about it?

File: lib/datadog.js [hunk 3]
```diff
@@ -4,39 +4,23 @@
 'use strict'
 
 require('loadenv')({ debugName: 'docker-listener' })
-var stream = require('stream')
-var util = require('util')
 var monitor = require('monitor-dog')
+var put = require('101/put')
 
-var log = require('./logger')()
+var baseName = 'docker-listener.'
```
Comment (podviaznikov) [inline lib/datadog.js]: you can set env `MONITOR_PREFIX= docker-listener`. So no need for this line and manual concat below

File: lib/datadog.js (hunk 2 above)
Comment (podviaznikov) [inline lib/datadog.js:22]: ok, I don't feel strongly

File: lib/datadog.js (hunk 3 above)
Comment (anandkumarpatel) [inline lib/datadog.js]: +1

**Comment by anandkumarpatel** [comment]
+1 @podviaznikov said good to merge

**Pull request merged**